### PR TITLE
Add archive notice to README

### PR DESCRIPTION
**Pull request by ssmall**

### CODE REVIEW

**Review by zsxking — DISMISSED** [review]

File: docs/README_sp.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 [English](../README.md) | **Español**
 
+## 🛑 Este proyecto ha sido archivado y no mantiene actualmente 🛑
```
Comment (josueetcom) [inline docs/README_sp.md:5]:
```suggestion
## 🛑 Este proyecto se ha archivado y no se mantiene actualmente 🛑
```

**Review by josueetcom — DISMISSED** [review]

File: docs/README_sp.md (hunk 1 above)
Comment (josueetcom) [reply docs/README_sp.md]: Could also do "está archivado" instead of "se ha archivado"

Comment (ssmall) [reply docs/README_sp.md]: Done

**Review by josueetcom — APPROVED** [review]

**Pull request merged**